src/pages/index.js

```diff
@@ -37,17 +52,21 @@ const addCardFormValidator = new FormValidator(
 
 addCardFormValidator.enableValidation();
 
+const avatarValidator = new FormValidator(avatarElement, validationSettings);
+
+avatarValidator.enableValidation();
+
 //Section
 
 const section = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: renderCard,
   },
   "#card-list"
 );
 
-section.renderItems();
+api.getInitialCards().then((cards) => section.renderItems(cards));
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**